### ISSUE TITLE
How can I customize the importer / exporter?

### ISSUE THREAD

**Issue by andychan94** [FriendsOfSylius/SyliusImportExportPlugin#242]
Hello. 
I want to add some fields to the importer and exporter, but I cannot figure out a way to extend & customize the plugin.
Are there any docs written about this anywhere? Or is there a PR I can refer to?
Thank you in advance!

**Comment by oallain** [comment]
Hello @andychan94 

Have you find your answer ?
In the Readme we have some informations about exporter, you can find it here https://github.com/FriendsOfSylius/SyliusImportExportPlugin#a-real-example
In example, exported fields are in third position of `arguments`.

Regards

**Comment by cdarken** [comment]
@oallain bumped into this requirement too. the instructions are clear enough, but I wanted to add support for Brand import, with loevgaard/sylius-brand-plugin.
If I understand correctly, I need to make my own product processor, right? But how do I add just a Brand retrieval and association like `setBrand` without rewriting your ProductProcessor class?

**Comment by cdarken** [comment]
Any suggestions here? Should I open a new issue?

**Comment by t-n-y** [comment]
Hey,

Any update on that issue? I want to add field to the product export, but i can't extend `ProductResourcePlugin` as it's a final class. Any other simple solution ?

**Comment by lorenzo-shots** [comment]
Yes, if it's final you can't extend `ProductResourcePlugin`, but you could _override_ it copying it in your `src/` dir, modify the code parts to your needs and redefining the vendor one in the `services.yaml` with your overridden `ProductResourcePlugin`.

**Comment by t-n-y** [comment]
@lorenzo-shots I know, my question was to avoid doing an override to every files i need to modify
